doc/smart-switch/high-availability/smart-switch-ha-dpu-scope-dpu-driven-setup.md

```diff
@@ -159,6 +181,18 @@ Here are how the workflows look like for the typical planned operations:
 
 ### 8.1. HA set creation
 
+The HA bring-up workflow is described below -
+  - The DPU starts out with it's initial HA scope Role as Dead.
+  - First the SDN controller pushes all configurations including the HA set and the HA scope with role set to Active but AdminState set to Disabled.
+  - Then the SDN controller starts the HA state-machine on the DPU by updating the HA scope AdminState to Enabled.
+  - DPU HA state transitions to Connecting and attempts to connect to its pair specified in the HA set.
+  - If the connection attempt is unsuccessful it moves to the InitializingToStandalone statei and waits for trigger from SDN controller.
+  - If the connection was successful then the DPU HA state transitions to Connected state and performs bulk sync to synchronize existing flows from the other DPU in the pair in case it is already Active.
+  - At the end of the bulk sync the HA state transitions to InitializingToActive and waits for trigger from SDN controller.
+  - At this point the DPU is fully synchronized with the pair and is ready to receive any traffic. But the DPU has not enabled its BFD session to the NPUs, so it will not recieve any data traffic yet.
+  - Whenever the SDN controller decides this DPU is ready to be online, it pushes the Activate-Role trigger.
```

Comment:
need to explain Activate-Role trigger and what it is for.

Reply:
added separate section